modulemd/v2/tests/ModulemdTests/translation_helpers.py

```diff
@@ -68,6 +68,52 @@ def test_translation_catalog_from_index(self):
         # There are 34 unique summaries and descriptions in f29.yaml
         self.assertEquals(len(catalog), 34)
 
+    def test_translations_from_catalog(self):
+        translation_files = ["../test_data/nl.po"]
+
+        catalogs = dict()
+        for f in translation_files:
+            with open(f, 'r') as infile:
+                catalog = pofile.read_po(infile)
+                self.assertTrue(catalog)
+
+                # Check if the catalog is internally consistent
+                _, err = catalog.check()
+                self.assertNone(err)
+                catalogs[catalog.locale_identifier] = catalog
+
+        # Create a Modulemd.ModuleIndex object containing the YAML stream
+        index = Modulemd.ModuleIndex.new()
+        yaml_path = "../test_data/f29.yaml"
+        ret, failures = index.update_from_file(yaml_path, True)
+        self.assertIsNotNone(index)
+
+        err = get_modulemd_translations_from_catalog(catalogs, index)
+        self.assertNone(err)
+
+        for locale, catalog in catalogs.items():
+            for msg in catalog:
+                locations = [x[0] for x in msg.locations]
+                for location in locations:
+                    props = location.split(";")
+
+                    module_name = props[0]
+                    module = index.get_module(module_name)
+                    self.assertIsNotNone(module)
+
+                    stream_name = props[1]
+                    stream = module.search_streams(stream_name, 0)[0]
+                    self.assertIsNotNone(stream)
+
+                    string_type = props[2]
+                    if string_type == "summary":
+                        assert stream.get_summary(locale) == msg.string
```

Comment:
```suggestion
                        self.assertEqual(stream.get_summary(), msg.string)
```
The `unittest.assertEqual()` function will show the diff if they don't match. It makes it easier to find the mistake.